diff --git a/harness/compactTestView.js b/harness/compactTestView.js
index 4dc6fb0..35c6b10 100644
--- a/harness/compactTestView.js
+++ b/harness/compactTestView.js
@@ -52,6 +52,7 @@ var compactTestView = (function() {
       this.addLink('Content Licenses', 'licenses.html');
       this.addLink('Media Coverage', 'coverage.html');
       this.addLink('Test Any Player', '/anyplayer');
+      this.addLink('HTML Tests', 'https://html5test.com');
       this.addTestSuites(testSuiteVersions[this.testSuiteVer].testSuites);
 
       for (var engineId in EngineVersions) {
