diff --git a/harness/compactTestView.js b/harness/compactTestView.js
index 21f062d..ddf3c9e 100644
--- a/harness/compactTestView.js
+++ b/harness/compactTestView.js
@@ -55,6 +55,7 @@ var compactTestView = (function() {
       this.addLink('HTML Tests', 'https://html5test.com');
       this.addLink('CSS Tests', 'https://css3test.com');
       this.addLink('JS ECMAScript Tests','https://compat-table.github.io/compat-table/es2016plus/');
+      this.addLink('WPT Tests', 'https://wpt.live/');
       this.addTestSuites(testSuiteVersions[this.testSuiteVer].testSuites);
 
       for (var engineId in EngineVersions) {
