From: Santhosh-Kumar-infosys <sakumar.ext@libertyglobal.com>
Date: Wed, 14 Jan 2026
Subject: Add application memory test

---
diff --git a/harness/compactTestView.js b/harness/compactTestView.js
index 604f4a9..5604be6 100644
--- a/harness/compactTestView.js
+++ b/harness/compactTestView.js
@@ -60,6 +60,7 @@ var compactTestView = (function() {
       this.addLink('System fonts', 'fonts.html');
       this.addLink('Mixed Content', 'https://googlesamples.github.io/web-fundamentals/fundamentals/security/prevent-mixed-content/active-mixed-content.html');
       this.addLink('GFX test', 'gfx-test.html');
+      this.addLink('Application memory test', 'application_memory.html');
 
       var platCfg = new XMLHttpRequest();
       platCfg.overrideMimeType("application/json");
