Index: MVT/js_mse_eme/harness/testView.js
===================================================================
--- MVT.orig/js_mse_eme/harness/testView.js
+++ MVT/js_mse_eme/harness/testView.js
@@ -130,6 +130,22 @@ function TestView(testSuiteVer, testView
 
     var outputArea = createElement('div', 'outputarea');
     var textArea = createElement('div', 'output');
+    textArea.setAttribute('tabindex', '0');
+    textArea.classList.add('focusable');
+    textArea.addEventListener('keydown', function(e) {
+
+    if (e.key === 'ArrowDown') {
+        textArea.scrollTop += 30;
+    }
+
+    if (e.key === 'ArrowUp') {
+
+        if (textArea.scrollTop > 0) {
+            textArea.scrollTop -= 30;
+            e.stopImmediatePropagation();
+        }
+    }
+    });
     var textAreaContainer = createElement('div', 'outputcontainer');
     textAreaContainer.appendChild(textArea);
     outputArea.appendChild(textAreaContainer);
Index: MVT/js_mse_eme/style.css
===================================================================
--- MVT.orig/js_mse_eme/style.css
+++ MVT/js_mse_eme/style.css
@@ -213,7 +213,8 @@ div.container_hidden {
 #output {
   height: 250px;
   width: 600px;
-  overflow: hidden;
+  overflow-y: auto;
+  overflow-x: auto;
   color: #369;
   border-width: 1px;
   border-style: solid;
